### PR TITLE
Kills the Vir Times and replaces the economic destination list with Polaris locations.

### PR DESCRIPTION
**Pull request by elgeonmb**
gibson gazzette can stay but you're on thin ice

### CODE REVIEW

File: code/modules/economy/TradeDestinations.dm [hunk 1]
```diff
@@ -16,85 +16,73 @@ var/list/weighted_mundaneevent_locations = list()
 /datum/trade_destination/proc/get_custom_eventstring(var/event_type)
 	return null
 
-//distance is measured in AU and co-relates to travel time
-/datum/trade_destination/centcomm
-	name = "CentCom"
-	description = "NanoTrasen's administrative centre for Tau Ceti."
+//distance is measured in Arbitrary and corelates to travel time, like, I guess
```
Comment (Belsima) [inline code/modules/economy/TradeDestinations.dm]: correlates, two Rs

File: code/modules/economy/TradeDestinations.dm [hunk 2]
```diff
@@ -16,85 +16,73 @@ var/list/weighted_mundaneevent_locations = list()
...
 	willing_to_sell = list()
 	viable_random_events = list(SECURITY_BREACH, CORPORATE_ATTACK, AI_LIBERATION)
 	viable_mundane_events = list(ELECTION, RESIGNATION, CELEBRITY_DEATH)
 
-/datum/trade_destination/anansi
-	name = "NSS Anansi"
-	description = "Medical station ran by Second Red Cross (but owned by NT) for handling emergency cases from nearby colonies."
+/datum/trade_destination/nohio
+	name = "New Ohio"
+	description = "A world in the Saggitarius Heights, home to Vey-Medical's research and development facilities."
```
Comment (Belsima) [inline code/modules/economy/TradeDestinations.dm]: Sagittarius has only one G

File: code/modules/economy/TradeDestinations.dm [hunk 3]
```diff
@@ -16,85 +16,73 @@ var/list/weighted_mundaneevent_locations = list()
...
 	willing_to_sell = list()
 	viable_random_events = list(RIOTS, INDUSTRIAL_ACCIDENT, BIOHAZARD_OUTBREAK, CULT_CELL_REVEALED, FESTIVAL, MOURNING)
 	viable_mundane_events = list(ELECTION, TOURISM, RESIGNATION)
 
-/datum/trade_destination/luthien
-	name = "Luthien"
-	description = "A small colony established on a feral, untamed world (largely jungle). Savages and wild beasts attack the outpost regularly, although NT maintains tight military control."
+/datum/trade_destination/nisp
+	name = "Nisp"
+	description = "A near-garden world known for its hostile wildlife and its NO2 atmosphere."
```
Comment (Belsima) [inline code/modules/economy/TradeDestinations.dm]: Nitrous oxide is N2O.

File: code/modules/economy/TradeDestinations.dm (hunk 1 above)
Comment (elgeonmb) [reply code/modules/economy/TradeDestinations.dm]: not fixing typos in comments on a file that's nigh-obsolete anyway

**Comment by Anewbe** [comment]
Tested?

**Comment by elgeonmb** [comment]
Yes.

**Pull request merged**